check_plugins_viewer_mandatory_fields.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+set -e
+
+FIELDS=("title" "publisher" "category" "icon" "description" "repository" "firstPublicationDate" "latestUpdateDate")
+CATEGORIES=("Editor" "Debugger" "Formatter" "Language" "Linter" "Snippet" "Theme" "Other")
+
+function check_field() {
+  if [[ $1 == "null" || $1 = "\'\'" ]];then
+    return 1;
+  fi
+  return 0
+}
+
+function check_category() {
```

Comment:
Could you add comments (aka docs) that describe function parameters and what it actually does? FYI it is unusual that function called check changes files.

Reply:
added some comments to functions